sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -106,292 +95,798 @@ impl BlobClient {
+    //         blob_name: self.blob_name.clone(),
+    //         container_name: self.container_name.clone(),
+    //         endpoint: self.endpoint.clone(),
+    //         pipeline: self.pipeline.clone(),
+    //         version: self.version.clone(),
+    //     }
+    // }
+
+    // /// Returns a new instance of BlockBlobClient.
+    // #[tracing::subclient]
+    // pub fn get_block_blob_client(&self) -> BlockBlobClient {
+    //     BlockBlobClient {
+    //         blob_name: self.blob_name.clone(),
+    //         container_name: self.container_name.clone(),
+    //         endpoint: self.endpoint.clone(),
+    //         pipeline: self.pipeline.clone(),
+    //         version: self.version.clone(),
+    //     }
+    // }
+
+    // /// Returns a new instance of PageBlobClient.
+    // #[tracing::subclient]
+    // pub fn get_page_blob_client(&self) -> PageBlobClient {
+    //     PageBlobClient {
+    //         blob_name: self.blob_name.clone(),
+    //         container_name: self.container_name.clone(),
+    //         endpoint: self.endpoint.clone(),
+    //         pipeline: self.pipeline.clone(),
+    //         version: self.version.clone(),
+    //     }
+    // }
+
+    /// The Get Properties operation returns all user-defined metadata, standard HTTP properties, and system properties for the
+    /// blob. It does not return the content of the blob.
     ///
     /// # Arguments
     ///
-    /// * `copy_source` - Specifies the name of the source page blob snapshot. This value is a URL of up to 2 KB in length that
-    ///   specifies a page blob snapshot. The value should be URL-encoded as it would appear in a request URI. The source blob must
-    ///   either be public or must be authenticated via a shared access signature.
     /// * `options` - Optional parameters for the request.
-    #[tracing::function("Storage.Blob.Container.Blob.copyFromUrl")]
-    pub async fn copy_from_url(
+    pub async fn get_properties(
         &self,
-        copy_source: String,
-        options: Option<BlobClientCopyFromUrlOptions<'_>>,
-    ) -> Result<Response<BlobClientCopyFromUrlResult, NoFormat>> {
+        options: Option<BlobClientGetPropertiesOptions<'_>>,
+    ) -> Result<Response<BlobClientGetPropertiesResult, NoFormat>> {
         let options = options.unwrap_or_default();
         let ctx = Context::with_context(&options.method_options.context);
-        let mut url = self.endpoint.clone();
-        let mut path = String::from("{containerName}/{blobName}");
-        path = path.replace("{blobName}", &self.blob_name);
-        path = path.replace("{containerName}", &self.container_name);
-        url = url.join(&path)?;
-        url.query_pairs_mut()
-            .append_pair("comp", "copy")
-            .append_key_only("sync");
+        let mut url = self.blob_url.clone();
+        if let Some(snapshot) = options.snapshot {
+            url.query_pairs_mut().append_pair("snapshot", &snapshot);
+        }
```

Comment:
This is the only currently working API

Generated code change #2:
On a per-API basis, we no longer need to get path parameters in there (i.e. container, blob) since it will now build off `blob_url` and not a naked `endpoint`